web/turkey/survey/static/survey/js/auditors/pastes_specific.js

```diff
@@ -0,0 +1,25 @@
+var AuditorPastesSpecific = {
+    pastes_specific: [],
+    log_paste_content: function (e) {
+        var pasted_data = e.clipboardData.getData('text');
+        this.pastes_specific.push(pasted_data);
+    },
+    submit_callable: function () {
+        return {
+            'content':  this.pastes_specific
```

Comment:
Same thing vis a vis Python here.

Suggestion:
Model:
paste_content = models.TextField()
timestamp = models.DateTimeField() # idk if we actually want this field